ourjapanlife/findadoc-frontend
# created logout page and functionality

[OLD] ## Resolves #55, resolves #56 

- [x] Added logout page
- [x] Login / Logout is now toggled in navbar based on logged in status. 
- [x] Implemented logout functionality  

### How to test

- If not logged in, navbar should say `Login`
- If logged in, navbar should say `Logout`
- Admin dashboard should not be accessible if logged out. Should redirect to login page
- Logout is Idempotent. can be repeated without consequence. (refresh the logout page)

### Screenshots

![Screenshot 2021-06-18 165852](https://user-images.githubusercontent.com/4411499/122527689-86aa3a80-d056-11eb-912b-4804a58a11a9.png)
